python/lsst/sims/maf/metrics/snSLMetric.py

```diff
@@ -1,19 +1,29 @@
 import numpy as np
 import lsst.sims.maf.metrics as metrics
 import healpy as hp
-from lsst.sims.maf.stackers import snStacker
-import numpy.lib.recfunctions as rf
+from .seasonMetrics import calcSeason
+from scipy.stats import binned_statistic
+from lsst.sims.utils import int_binned_stat
+from lsst.sims.photUtils import Dust_values
 
 __all__ = ['SNSLMetric']
 
 
+def coaddM5(mags):
+    """Coadded depth, assuming Gaussian noise
+    """
+    return 1.25 * np.log10(np.sum(10.**(0.8*np.array(mags))))
```

Comment:
Isn't this just the CoaddM5Metric?

Reply:
Yup, but need it as a simple function so scipy things can call it as a reduceFunc.